src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3691,18 +3691,28 @@
      * <p>For example:</p>
      *
      * <pre>
-     *     A extends Collection
+     * class MyClass &lt;A extends Collection&gt; {
+     *
+     * }
      * </pre>
      *
      * <p>parses as:</p>
      *
      * <pre>
-     * +--TYPE_PARAMETER
-     *     |
-     *     +--IDENT (A)
-     *     +--TYPE_UPPER_BOUNDS
-     *         |
-     *         +--IDENT (Collection)
+     * CLASS_DEF -&gt; CLASS_DEF
+     * |--MODIFIERS -&gt; MODIFIERS
```

Comment:
Please add `public` modifier to the class definition to avoid empty `MODIFIERS` node in the tree. It is a bit confusing.

Reply:
done